ui/raidboss/data/06-ew/trial/hydaelyn-ex.txt

```diff
@@ -49,27 +57,33 @@ hideall "--sync--"
 288.4 "--sync--" sync / 1[56]:[^:]*:Hydaelyn:65A0:/
 289.2 "Crystalline Blizzard III" sync / 1[56]:[^:]*:Hydaelyn:65A4:/
 296.9 "Magos's Radiance" sync / 1[56]:[^:]*:Hydaelyn:65C2:/
-307.1 "Aureole/Lateral Aureole" sync / 1[56]:[^:]*:Hydaelyn:(6C91|6F13):/
+# -> naturally into Crystallize 3 (or jump from above)
 
 # Crystallize 3 (Equinox and Water/Stone)
+307.1 "Aureole/Lateral Aureole" sync / 1[56]:[^:]*:Hydaelyn:(6C91|6F13):/
 316.4 "Crystallize" sync / 1[56]:[^:]*:Hydaelyn:(659E|6ED5|659A|659B):/
 325.6 "Equinox" sync / 1[56]:[^:]*:Hydaelyn:65C3:/
 328.7 "--sync--" sync / 1[56]:[^:]*:Hydaelyn:659F:/
 329.6 "Crystalline Water/Stone III" sync / 1[56]:[^:]*:Hydaelyn:65A[23]:/
+331.7 "--untargetable--"
+
 
 # Adds
-336.2 "--transition--" sync / 1[56]:[^:]*:Hydaelyn:65BC:/ window 300,10
+336.2 "--sync--" sync / 1[56]:[^:]*:Hydaelyn:65BC:/ window 350,10
 344.3 "Pure Crystal" sync / 1[56]:[^:]*:Hydaelyn:65BD:/
+344.3 "--targetable--"
 
-1433.6 "Exodus" sync / 1[56]:[^:]*:Hydaelyn:65BB:/ window 1100,10
+1433.6 "Exodus" sync / 1[56]:[^:]*:Hydaelyn:65BB:/ window 2000,10
 1448.4 "Exodus" sync / 1[56]:[^:]*:Hydaelyn:662B:/
+1453.7 "--targetable--"
 1468.8 "Halo" sync / 1[56]:[^:]*:Hydaelyn:65A5:/
 
+
 # Lightwave 1
 1476.9 "Lightwave" sync / 1[56]:[^:]*:Hydaelyn:6693:/
-1488.7 "Light of the Crystal" sync / 1[56]:[^:]*:Hydaelyn:65A6:/
+1488.7 "Light of the Crystal" #sync / 1[56]:[^:]*:Hydaelyn:65A6:/
```

Comment:
These were just too close in time, and `test_timeline` showed that the second one would resync.

Reply:
Thanks for testing this.